Gemfile.lock

```diff
@@ -230,5 +234,8 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   web-console
 
+RUBY VERSION
+   ruby 2.3.1p112
+
```

Comment:
Do we need this? Is this something that should have been added when the app was created?